packages/nc-gui/components/cell/PhoneNumber.vue

```diff
@@ -1,21 +1,84 @@
-<script setup lang="ts">
-import { useVModel } from '#imports'
+<script lang="ts" setup>
+import type { VNodeRef } from '@vue/runtime-core'
+import isMobilePhone from 'validator/lib/isMobilePhone'
+import { EditColumnInj, EditModeInj, IsExpandedFormOpenInj, IsSurveyFormInj, computed, inject } from '#imports'
 
 interface Props {
-  modelValue: any
+  modelValue: string | null | number | undefined
 }
 
-interface Emits {
-  (event: 'update:modelValue', model: string): void
-}
+const { modelValue: value } = defineProps<Props>()
```

Comment:
`value` won't be reactive, use `toRef` or `useVModel`

Reply:
We just use the initial value